### PR TITLE
DW-4009 Added Lambda to check for user expiry

### PR DESCRIPTION
**Pull request by fernando-schmidt-dmw**

### CODE REVIEW

File: terraform/modules/pre-auth-lambda/outputs.tf [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
   value = aws_lambda_function.lambda_pre_auth
 }
+
+output dynamodb_table_user_arn {
+  value = aws_dynamodb_table.dynamodb_table_user.arn
+}
+
+output dynamodb_table_user_name {
+  value = aws_dynamodb_table.dynamodb_table_user.name
+}
```
Comment (dan-hill2802) [inline terraform/modules/pre-auth-lambda/outputs.tf:11]:
```suggestion
output dynamodb_table_user {
  value = {
      arn = aws_dynamodb_table.dynamodb_table_user.arn
      name = aws_dynamodb_table.dynamodb_table_user.name
    }
}
```

Comment (dan-hill2802) [reply terraform/modules/pre-auth-lambda/outputs.tf:11]: formatting is a little off there due to editing in comment box, but `terraform fmt` will fix

Comment (fernando-schmidt-dmw) [reply terraform/modules/pre-auth-lambda/outputs.tf]: This is done thanks.

File: terraform/modules/check-user-expiry-lambda/s3_objects/email_from.txt.j2 [hunk 2]
```diff
@@ -0,0 +1 @@
+DataWorks Access Management <{{email_address_dataworks_access_management}}>
```
Comment (dan-hill2802) [inline terraform/modules/check-user-expiry-lambda/s3_objects/email_from.txt.j2:1]: this needs to be sourced from `aws-common-infrastructure` remote state

Comment (fernando-schmidt-dmw) [reply terraform/modules/check-user-expiry-lambda/s3_objects/email_from.txt.j2]: This is done, thanks.

**Pull request closed without merging**